docs/_data/toc.yml

```diff
@@ -159,6 +159,8 @@
       url: guides/troubleshooting
     - title: Upgrading Meshery
       url: guides/upgrade
+    - title: Send Events From Meshery Server Using Golang
```

Comment:
Move this to the Contributing section, please.

Rename to follow the Contributing Doc page naming convention.